connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -1011,5 +1039,7 @@ service SparkConnectService {
 
   // FetchErrorDetails retrieves the matched exception with details based on a provided error id.
   rpc FetchErrorDetails(FetchErrorDetailsRequest) returns (FetchErrorDetailsResponse) {}
-}
 
+  // Build ResourceProfile and get the profile id
+  rpc BuildResourceProfile(BuildResourceProfileRequest) returns (BuildResourceProfileResponse) {}
```

Comment:
Why does this need to be an extra RPC and not just a command?

Reply:
Hi @grundprinzip, Really good suggestion, just made the newest commit to move it to the command.